sass/_main.scss

```diff
@@ -537,14 +542,14 @@ hr {
 .vid-insert-wrap {
   position: relative;
   width: 100vw;
-  height: 850px;
+  height: 100vh;
```

Comment:
Does this video style change break the media embeds on `/media` or its posts? (eg. `/media/blueprint-2017-galen-wolfe-pauly-on-the-distributed-web/`)

Reply:
Nope, the front page vid is the only consumer of this